src/pages/spawn/spawnVolume/spawnVolumeTableActions/MigrateButton.tsx

```diff
@@ -4,7 +4,7 @@ import { GuideCue } from "@leafygreen-ui/guide-cue";
 import Cookies from "js-cookie";
 import { SEEN_MIGRATE_GUIDE_CUE } from "constants/cookies";
 import { TableVolume } from "types/spawn";
-import { MigrateVolumeModal } from "./migrateButton/MigrateVolumeModal";
+import { MigrateVolumeModal } from "./MigrateVolumeModal";
```

Comment:
Not sure if this already has a ticket but if not we should cut one to migrate it as well.